src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/service/GroupsServiceTest.kt

```diff
@@ -67,4 +71,40 @@ class GroupsServiceTest {
       null
     )
   }
+
+  @Nested
+  inner class createGroup {
+    @BeforeEach
+    fun initSecurityContext() {
+
+      whenever(authenticationFacade.currentUsername).thenReturn("username")
+      SecurityContextHolder.getContext().authentication = authentication
```

Comment:
Do you need this line? - SecurityContextHolder.getContext().authentication = authentication